block/adapter.go

```diff
@@ -56,6 +56,7 @@ type Adapter interface {
 	GetRange(obj ObjectPointer, startPosition int64, endPosition int64) (io.ReadCloser, error)
 	GetProperties(obj ObjectPointer) (Properties, error)
 	Remove(obj ObjectPointer) error
+	Copy(sourceObj, destinationObj ObjectPointer) error
```

Comment:
Don't these need to be `ObjectPointer`s?  Otherwise how do we handle namespaces?

Reply:
They are `ObjectPointer`s

Reply:
Aaah, evidently I can copy but I cannot read.